R/system.R

```diff
@@ -300,7 +300,7 @@ getMongoCollectionNames <- function(host = "", port = "", database = "", usernam
 getMongoCollectionNumberOfRows <- function(host = NULL, port = "", database = "",
                                            username = "", password = "", collection = "",
                                            isSSL=FALSE, authSource=NULL, cluster = NULL, additionalParams = "",
-                                           timeout = timeout, ...){
+                                           timeout = "", ...){
```

Comment:
Why does it have to be an empty string as opposed to NULL, which seems like more natural choice provided this is a numeric argument?

Reply:
Updated as suggested.